kong/dao/migrations/cassandra.lua

```diff
@@ -423,4 +423,27 @@ return {
     end,
     down = function(_, _, dao) end
   },
+  {
+    name = "2017-07-31-120200_key-auth_preflight_default",
+    up = function(_, _, dao)
+      local rows, err = dao.db:query([[
+        SELECT * FROM plugins WHERE name = 'key-auth';
```

Comment:
Additionally, this migration does not belong to the core, but to the key-auth plugin.